xarray/tests/test_interp.py

```diff
@@ -462,19 +462,30 @@ def test_interp_like():
 
 
 @requires_scipy
-def test_datetime():
-    da = xr.DataArray(np.random.randn(24), dims='time',
+@pytest.mark.parametrize('x_new, expected', [
```

Comment:
I think `requires_scipy` does not work with another decorator. (We need to improve it though)

A workaround is to `pytest.importorskip('scipy')` *inside* the function.

Reply:
OK. Forget my comment. I will be merging it later.